hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -115,6 +117,23 @@ def to_json(self) -> t.Dict:
         return asdict(self)
 
 
+def record_content_submission(
+    dynamodb_table: Table, image_folder_key: str, request: SubmitContentRequestBody
+):
+    # TODO add a confirm overwrite path for this
+    submit_time = datetime.datetime.now()
+    ContentObject(
+        content_id=request.content_id,
+        content_type="PHOTO",
+        content_ref=f"{image_folder_key}{request.content_id}",  # raw bytes + tmp urls are a bad idea atm, assume s3 object for now
+        content_ref_type=request.submission_type,
+        additional_fields=set(request.metadata) if request.metadata else set(),
+        submissions=[submit_time],
```

Comment:
Implies re-submissions overwrite. Please confirm.

Reply:
Does imply but I handled that with 
`SubmissionTimes = list_append(if_not_exists(SubmissionTimes, :empty_list), :s)` in `UpdateExpression`
meaning all new values are appended. 

This gets messy if we start allowing query then update (in which case we'd want to null out submission or do something else fancy.) 

Will add comment.